browser-test/src/application_filter.test.ts

```diff
@@ -0,0 +1,149 @@
+import {
+  createTestContext,
+  dropTables,
+  loginAsAdmin,
+  loginAsProgramAdmin,
+  loginAsTestUser,
+  logout,
+  seedQuestions,
+  validateScreenshot,
+} from './support'
+
+describe('normal application flow', () => {
+  const ctx = createTestContext()
+
+  beforeAll(async () => {
+    const {page} = ctx
+    await dropTables(page)
+    await seedQuestions(page)
+  })
+
+  it('all major steps', async () => {
```

Comment:
**suggestion (blocking):** update the strings passed to `describe` and `it` to summarize the user behavior exercised by this test